mbuild/packing.py

```diff
@@ -171,6 +179,9 @@ def fill_box(
         default=False.
     temp_file : str, default=None
         File name to write PACKMOL raw output to.
+    save_packmol_input : bool, default=False,
```

Comment:
What if instead you passed a filename here, and call the argument "packmol_filename". Such that if you pass a specific filename, that's what would save. Might make it even easier to debug if you're running a loop of molecules to check which are failing, so you don't continually override that file. If this argument is None, then it doesn't save to disk.

Reply:
This should be resolved. LMK what you think.